documentation/docs/developers/strategies/simple-trade.md

```diff
@@ -0,0 +1,12 @@
+# Simple Trade
+
+## Overview
+
+Simple trade strategy expands upon Perform Trade strategy. It has a few additional features such as time constraints and order specific loggings.
+
+#### Time restriction
+There is a time restriction `_time_delay` set that will ensure that there is a given amount of time in between the trades. When an order is placed in `c_process_market()`, it checks that the current timestamp is greater than previous order’s timestamp plus delay time. If it is a valid time to submit new order, the order will be processed. 
+There is also a restriction `_cancel_order_wait_time` placed to cancel active orders. If the current timestamp is greater than order’s start time plus cancel wait time, the order is cancelled.
+
+#### Logging
+Helpful logging about order placement status is provided. The functions `c_did_fill_order()`, `c_did_complete_buy_order()`, and `c_did_complete_sell_order()` will log when order is either filled or completed.
```

Comment:
And again... it's better to describe this in terms of a new mechanism that the reader can use to develop his/her own strategies. In this case, it's the event handling mechanism.

When you go into the details of what each of these functions is doing, it may be better to use point form as well.